eclass/distutils-r1.eclass

```diff
@@ -1854,6 +1915,10 @@ _distutils-r1_check_namespace_pth() {
 		ewarn "the ebuild accordingly:"
 		ewarn
 		ewarn "  https://projects.gentoo.org/python/guide/concept.html#namespace-packages"
+
+		if ! has "${EAPI}" 5 6 7 8; then
```

Comment:
I think `EAPI=5` is already banned for this eclass?

Reply:
You're right.